integration/use-cases/create-private-registry.js

```diff
@@ -2,53 +2,68 @@ const axios = require("axios");
 const utils = require("./lib/utils");
 
 test("Creates a private registry", async () => {
-  await page.goto(getUrl("/#/c/default/ns/default/config/repos"));
-
-  await expect(page).toFillForm("form", {
-    token: process.env.ADMIN_TOKEN,
+  var token = process.env.USE_MULTICLUSTER_OIDC_ENV ? undefined : process.env.ADMIN_TOKEN;
+  page.on("response", response => {
+    // retrieves the token after the oidc flow, note this require "--set-authorization-header=true" flag to be enabled in oauth2proxy
+    token = response.headers()["authorization"] || token;
   });
 
-  await page.evaluate(() =>
-    document.querySelector("#login-submit-button").click()
+  await utils.login(
+    page,
+    process.env.USE_MULTICLUSTER_OIDC_ENV,
+    "/#/c/default/ns/default/config/repos",
+    process.env.ADMIN_TOKEN,
+    "kubeapps-operator@example.com",
+    "password",
   );
 
+  // wait for the loading msg to disappear
+  await page.waitForFunction(() => !document.querySelector("cds-progress-circle"));
```

Comment:
unless there is a reason that waiting for the button doesn't work I won't add these kind of waits, it just makes the tests more difficult to maintain.